core/src/main/scala/kafka/tools/ExportZkOffsets.scala

```diff
@@ -1,132 +0,0 @@
-/**
```

Comment:
Can we also remove ImportZkOffsets?   

Also we have not removed ZK based offset management.  Are are going to support ZK based offset management some more time?  Existing users can continue to use older release scala clients if they want?

Can we deprecate ZK based offset management ?

Reply:
Good point, removed `ImportZkOffsets`. What do you mean by "we have not removed ZK based offset management?

Reply:
I was mentioning about below code segments. 
https://github.com/omkreddy/kafka/blob/trunk/core/src/main/scala/kafka/server/KafkaApis.scala#L322
https://github.com/omkreddy/kafka/blob/trunk/core/src/main/scala/kafka/zk/ZkData.scala#L428

Reply:
Yeah, we want the broker to still support everything for now. We need a KIP to discuss removal and deprecation of broker functionality related to the old consumer. I restored a change in `ZkData` as I hadn't realised that the broker was relying on that functionality itself.